index.js

```diff
@@ -0,0 +1,53 @@
+var express = require('express');
+var app = express();
+app.set('view engine', 'ejs');
+var bodyParser = require('body-parser')
+
+app.set('view engine', 'ejs');
+app.use(bodyParser.urlencoded({extended: false}));
+
+app.use(express.static(__dirname + '/views'));
```

Comment:
This line of code here sets our static folder, so in your case, we want the path to be '/static':
```
app.use(express.static(__dirname + '/static'));
```